doc/book/admin/access_control.rst

```diff
@@ -1,270 +1,854 @@
+
+
+In this example, the password is specified in the ``options`` parameter:
+
+..  literalinclude:: /code_snippets/test/access_control/grant_user_privileges_test.lua
+    :language: lua
+    :start-after: Create a user with a password
+    :end-before: End: Create a user with a password
+    :dedent:
+
+
+.. _access_control_user_changing_passwords:
+
+Changing passwords
+~~~~~~~~~~~~~~~~~~
+
+To set or change a user's password, use :ref:`box.schema.user.passwd() <box_schema-user_create>`.
+In the example below, a user password is set for a :ref:`currently <access_control_users_current_user>` logged-in user:
+
+..  literalinclude:: /code_snippets/test/access_control/grant_user_privileges_test.lua
+    :language: lua
+    :start-after: Set a password for the current user
+    :end-before: End: Set a password for the current user
+    :dedent:
+
+To set the password for the specified user, pass a username and password as shown below:
+
+..  literalinclude:: /code_snippets/test/access_control/grant_user_privileges_test.lua
+    :language: lua
+    :start-after: Set a password for the specified user
+    :end-before: End: Set a password for the specified user
+    :dedent:
+
+..  NOTE::
+
+    :doc:`/reference/reference_lua/box_schema/user_password` returns a hash of the specified password.
+
+
+
+.. _access_control_user_granting_privileges:
+
+Granting privileges to a user
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To grant the specified privileges to a user, use the ``box.schema.user.grant()`` function.
+In the example below, 'testuser' gets read privileges to the ``writers`` space and read/write privileges to the ``books`` space:
+
+..  literalinclude:: /code_snippets/test/access_control/grant_user_privileges_test.lua
+    :language: lua
+    :start-after: Grant privileges to the specified user
+    :end-before: End: Grant privileges to the specified user
+    :dedent:
+
+Learn more about granting privileges to different types of objects from :ref:`access_control_granting_privileges`.
+
+
+
+.. _access_control_user_info:
+
+Getting a user's information
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To check whether the specified user exists, call :ref:`box.schema.user.exists() <box_schema-user_exists>`:
+
+..  code-block:: lua
+
+    box.schema.user.exists('testuser')
+    --[[
+    - true
+    --]]
+
+To get information about privileges granted to a user, call :ref:`box.schema.user.info() <box_schema-user_info>`:
+
+..  code-block:: lua
+
+    box.schema.user.info('testuser')
+    --[[
+    - - - execute
+        - role
+        - public
+      - - read
+        - space
+        - writers
+      - - read,write
+        - space
+        - books
+      - - session,usage
+        - universe
+        -
+      - - alter
+        - user
+        - testuser
+    --]]
+
+In the example above, 'testuser' has the following privileges:
+
+*   The 'execute' privilege to the 'public' role means that this role is :ref:`assigned to a user <access_control_roles_granting_user>`.
```

Comment:
> assigned to the user

Exactly! Thanks for the catch

Reply:
> Maybe note here, that 'public' is granted to all users on creation?
(I know, that this is stated in the Roles section, but it could be helpful in describing a users privileges too)

I'd keep it as is because it is a how-to section and maintaining such info in several places might be hard if some privileges/roles change for the default user.

Reply:
> Exactly! Thanks for the catch

It seems like there is the same issue in other bullet-points below

Reply:
> It seems like there is the same issue in other bullet-points below

Thanks, fixed